mimic/canned_responses/images_json.py

```diff
@@ -0,0 +1,51 @@
+"""
+Glance images
+"""
+
+images = {
+    "images":
+        {
```

Comment:
This should be a list; i.e.

`"images": {...}`

should be

`"images": [{...}]`.